libs/src/sdk/sdk.ts

```diff
@@ -83,13 +83,22 @@ type SdkConfig = {
    * API secret, required for writes
    */
   apiSecret?: string
+
+  /**
+   * Use selected discovery provider for writes
+   */
+  useDiscoveryRelay?: boolean
```

Comment:
I'd leave this as a config for entity manager rather than top level - if people want to use it they can specify an EntityManager instance